src/olympia/pages/templates/pages/shield_study_13.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Understanding the Web</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_13/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Understanding the Web</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/832183/addon-832183-latest.xpi"><button class="button primary large">Start Now</button></a>
```

Comment:
This links to an unsigned add-on apparently? At least I'm unable to install it in nightly, is this expected?